test/unit/property-effects-template.html

```diff
@@ -162,6 +162,30 @@
 </script>
 </dom-module>
 
+<script type="module">
+  import { PolymerElement, html } from '../../polymer-element.js';
+  customElements.define('x-runtime-nested', class extends PolymerElement {
+    static get template() {
+      return html`<template id="t1">
```

Comment:
Let's rename these to reflect the nesting.

Reply:
Done.